examples/tutorial/tutorial/monitors.py

```diff
@@ -53,4 +53,4 @@ class PeriodicMonitorSuite(MonitorSuite):
 class SpiderCloseMonitorSuite(MonitorSuite):
     monitors = [ItemCountMonitor, ItemValidationMonitor, PeriodicJobStatsMonitor]
 
-    monitors_failed_actions = [SendSlackMessageSpiderFinished]
+    monitors_failed_actions = [SendTelegramMessageSpiderFinished]
```

Comment:
I don't think we should change our tutorial example code with this new action. The purpose of the tutorial is just to show some possibilities, but not to use every possible feature of Spidermon.

Reply:
Yes @rennerocha I was just testing it and commited to simplify my organization, I already removed this change on another commit.